### PR TITLE
fix binary check for cephfs.go

### LINKED ISSUE #56479: exec.Run didn't return an error when ls has a non-zero exit code
Some code uses `ls` to check if binary exist. But it does not work as expected.
use `test -x file`  instead.
/kind bug

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
unify the check method. see ref issue： #56479

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #56479

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
@rootfs PTAL

File: pkg/volume/cephfs/cephfs.go [hunk 1]
```diff
@@ -331,9 +331,8 @@ func (cephfsMounter *cephfsMounter) checkFuseMount() bool {
 	execute := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
 	switch runtime.GOOS {
 	case "linux":
-		retBytes, err := execute.Run("/bin/ls", "/sbin/mount.fuse.ceph")
-		if err == nil && string(retBytes) == "/sbin/mount.fuse.ceph\n" {
-			glog.V(4).Infof("/sbin/mount.fuse.ceph exists, it should be fuse mount")
+		if _, err := execute.Run("test", "-x", "/sbin/mount.fuse.ceph"); err == nil {
```
Comment (rootfs) [inline pkg/volume/cephfs/cephfs.go]: ls and test are both with coreutils. If ls is missing, so is test.

Comment (zhangxiaoyu-zidif) [reply pkg/volume/cephfs/cephfs.go]: we have testify them both.  `test` works well， but `ls` does not. We do not know why. we have some discuss before in https://github.com/kubernetes/kubernetes/pull/55992

Comment (rootfs) [reply pkg/volume/cephfs/cephfs.go]: which linux distribution you are using?

Comment (zhangxiaoyu-zidif) [reply pkg/volume/cephfs/cephfs.go]: We are using linux distribution based on CentOS 7.3

Comment (rootfs) [reply pkg/volume/cephfs/cephfs.go]: I suspect there is a `ls` alias somewhere in some shell profiles.

Just to be precise, use abs path `/usr/bin/test`

Comment (zhangxiaoyu-zidif) [reply pkg/volume/cephfs/cephfs.go]: thanks very much. you are right~ =)

**Comment by zhangxiaoyu-zidif** [comment]
@rootfs 
PTAL

**Pull request merged**